TODO FIXME: remove "testedfiles" patches once include
files works as expected

diff --git a/agents/Makefile.am b/agents/Makefile.am
index c09e509..e69de29 100644
--- a/agents/Makefile.am
+++ b/agents/Makefile.am
@@ -1,7 +0,0 @@
-#
-# Makefile.am for backup/agents
-#
-
-agent_SCRIPTS =		ag_file_append
-
-EXTRA_DIST = $(agent_SCRIPTS) $(scrconf_DATA)
diff --git a/doc/autodocs/Makefile.am b/doc/autodocs/Makefile.am
index 2a6f678..9de7be0 100644
--- a/doc/autodocs/Makefile.am
+++ b/doc/autodocs/Makefile.am
@@ -1,3 +1,5 @@
 # Makefile.am for YCP module .../doc/autodocs
 
-include $(top_srcdir)/autodocs-ycp.ami
+# TODO FIXME: temporarily disabled
+
+#include $(top_srcdir)/autodocs-ycp.ami
diff --git a/testsuite/tests/Ext2Filesystems.ycp b/testsuite/tests/Ext2Filesystems.ycp
index 646e4ae..11a6b99 100644
--- a/testsuite/tests/Ext2Filesystems.ycp
+++ b/testsuite/tests/Ext2Filesystems.ycp
@@ -2,7 +2,7 @@
 //
 // Ext2Filesystems function tests
 //
-// testedfiles: backup/functions.ycp
+// testedfiles: backup/functions.ycp tests/Ext2Filesystems.rb
 //
 // $Id$
 //
diff --git a/testsuite/tests/Ext2MountPoint.ycp b/testsuite/tests/Ext2MountPoint.ycp
index 0e15277..6e2ded3 100644
--- a/testsuite/tests/Ext2MountPoint.ycp
+++ b/testsuite/tests/Ext2MountPoint.ycp
@@ -2,7 +2,7 @@
 //
 // Ext2MountPoint function tests
 //
-// testedfiles: backup/functions.ycp
+// testedfiles: backup/functions.ycp tests/Ext2MountPoint.rb
 //
 // $Id$
 //
diff --git a/testsuite/tests/GetMountedFilesystems.ycp b/testsuite/tests/GetMountedFilesystems.ycp
index 1a0f628..27ee7a8 100644
--- a/testsuite/tests/GetMountedFilesystems.ycp
+++ b/testsuite/tests/GetMountedFilesystems.ycp
@@ -2,7 +2,7 @@
 //
 // GetMountedFilesystems function tests
 //
-// testedfiles: backup/functions.ycp
+// testedfiles: backup/functions.ycp tests/GetMountedFilesystems.rb
 //
 // $Id$
 //
diff --git a/yast2-backup.spec.in b/yast2-backup.spec.in
index 3f5b8fc..6be0439 100644
--- a/yast2-backup.spec.in
+++ b/yast2-backup.spec.in
@@ -4,6 +4,8 @@
 Group:          System/YaST
 License:        GPL-2.0
 
+Requires:       yast2-ruby-bindings >= 1.0.0
+
 Summary:		YaST2 - System Backup
 BuildArchitectures:	noarch
 
@@ -65,10 +67,9 @@ backs them up.
 @agentdir@/ag_file_append
 @scrconfdir@/cfg_backup.scr
 @yncludedir@/backup/*
-@clientdir@/backup*.ycp
+@clientdir@/backup*.rb
 @ybindir@/backup_*
-@moduledir@/Backup.ycp
-@moduledir@/Backup.ybc
+@moduledir@/Backup.rb
 @desktopdir@/backup.desktop
 %doc @docdir@
 
